db/column_family.cc

```diff
@@ -342,6 +342,13 @@ ColumnFamilyOptions SanitizeOptions(const ImmutableDBOptions& db_options,
     result.max_compaction_bytes = result.target_file_size_base * 25;
   }
 
+  if (result.compaction_style == kCompactionStyleLevel &&
+      (result.compaction_filter != nullptr ||
+       result.compaction_filter_factory != nullptr) &&
+      result.periodic_compaction_seconds == 0) {
```

Comment:
Does 0 mean disabled? If 0 means disabled, and we sanitize it then there is no way users to explicitly disable it anymore. This is something we should fix.

Reply:
Yeah, 0 means it is disabled. 
`periodic_compaction_seconds` is a uint64_t type. So, if someone really wants to explicitly disable periodic compactions, they could just set the value to UINT64_MAX. What do you say?

Reply:
I know there is no ideal solution here. We can go both ways. 0 means disable, UINT64_MAX means something that can be sanitized to be enabled or not, which would be the default. The benefit with this solution is that, they still get the same behavior if they have explicitly set it to be 0 for now. Or, another thing to consider is to use UINT64_MAX - 1 as the default and the special flag for a value to be sanitized, which hopefully won't impact anyone for now.

Reply:
Changed the default to UINT64_MAX. Thanks for the suggestion.